imager/imagerapp/tests.py

```diff
@@ -61,3 +57,96 @@ def test_unicode_and_str(self):
         bill_unicode = unicode(bill.profile)
         self.assertEqual(isinstance(bill_str, str), True)
         self.assertEqual(isinstance(bill_unicode, unicode), True)
+
+
+class ImagerFollowTestCase(TestCase):
+    def setUp(self):
+        self.bill = User(username='bill')
+        self.sally = User(username='sally')
+        self.tracy = User(username='tracy')
+        self.bill.save()
+        self.sally.save()
+        self.tracy.save()
```

Comment:
What about something like this:

```python
def setUp(sefl):
    self.users = {}
    for username in ['sally', 'bill', 'tracy']:
        user = User(username=username)
        user.save()
        self.users[username] = user
```

Then you can refer to `self.users['bill']` in tests.  A bit cleaner than what you have now.